roles/nginxplus/files/conf/http/lae-staging.conf

```diff
@@ -13,11 +13,14 @@ limit_req_zone $external_traffic zone=lae-staging-ratelimit:10m rate=10r/s;
 
 upstream lae-staging {
     zone lae-staging 64k;
-    server lae-staging2.princeton.edu resolve;
+    server service.consul service=node-0.traefik-wall-staging resolve max_fails=0;
+    server service.consul service=traefik-wall-staging resolve max_fails=0 backup;
```

Comment:
This forwards all requests through one node unless it's down. Another option would be to use the ip_hash algorithm instead, so a given IP lands on a given node.

Reply:
We could also put all the regular passenger servers here as backup nodes so if Nomad crashes the sites still work, they just lose turnstile.